review-me/review-me.js

```diff
@@ -0,0 +1,26 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
```

Comment:
11  :  missing  sem icolon  after  console .log (" Nothing  to  print "),  code  style  issue